k8s/compose/docker-compose.yml

```diff
@@ -5,110 +5,100 @@ services:
     build:
       context: ../..
       dockerfile: k8s/Dockerfile
-    network_mode: host
-    restart: always
     command: [ "/usr/local/bin/block-oracle", "/app/config.toml" ]
-    depends_on:
-      epoch-subgraph-check:
-        condition: service_completed_successfully
     environment:
       RUST_BACKTRACE: '1'
-      SUBGRAPH_URL: 'http://127.0.0.1:8000/subgraphs/name/edgeandnode/block-oracle'
+      SUBGRAPH_URL: 'http://graph-node:8000/subgraphs/name/edgeandnode/block-oracle'
       DATA_EDGE_CONTRACT_ADDRESS: 'e78a0f7e598cc8b0bb87894b0f60dd2a88d6a8ab'
-      EPOCH_MANAGER_CONTRACT_ADDRESS: 'D833215cBcc3f914bD1C9ece3EE7BF8B14f841bb'
+      EPOCH_MANAGER_CONTRACT_ADDRESS: 'd833215cbcc3f914bd1c9ece3ee7bf8b14f841bb'
+      PROTOCOL_CHAIN_JRPC_URL: "http://hardhat:8545"
     volumes:
       - ./config.toml:/app/config.toml
+    depends_on:
+      graph-node:
+        condition: service_healthy
+      epoch-subgraph:
+        condition: service_completed_successfully
 
   graph-node:
     image: graphprotocol/graph-node
-    network_mode: host
-    restart: always
     environment:
-      ipfs: '127.0.0.1:5001'
-      postgres_host: localhost
+      ipfs: 'ipfs:5001'
+      postgres_host: postgres
       postgres_port: '5432'
       postgres_user: graph-node
       postgres_db: graph-node
       postgres_pass: let-me-in
-      ethereum: 'hardhat:http://localhost:8545'
+      ethereum: 'hardhat:http://hardhat:8545'
       GRAPH_LOG: debug
+    ports:
+      - 8020
+      - 8000
+    healthcheck:
+      test: nc -z localhost 8020 || exit 1
+      interval: 10s
+      timeout: 5s
+      start_period: 10s
```

Comment:
We might want to remove some of these options, or maybe even all, and fallback to defaults. Just a suggestion, leaving it up to you.

Reply:
Same for the other healthchecks.

Reply:
Agree. Fixed in d315e5b.